pkgs/development/libraries/nanopb/default.nix

```
@@ -6,6 +6,7 @@
 , python3
 , stdenv
 , buildPackages
+, mallocBuild ? false  # Required by Criterion
```

```suggestion
, mallocBuild ? false
```
that comment will just bitrot over time.